### PR TITLE
Fix the preoblem when using select into with view path which is same as base path.

### PR DESCRIPTION
**Pull request by Beyyes**
## Description

The situation is as below, when the source columns have both base and view paths, and all these paths indicate same base paths.

a4746553-4455-45b0-a2ed-59d5a2006b12

image

### CODE REVIEW

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/operator/process/IntoOperator.java [hunk 1]
```diff
@@ -42,6 +44,8 @@ public class IntoOperator extends AbstractIntoOperator {
 
   private final List<Pair<String, PartialPath>> sourceTargetPathPairList;
 
+  private final List<Pair<String, String>> sourceColumnToViewList;
```
Comment (JackieTien97) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/execution/operator/process/IntoOperator.java:47]: It seems that `List<String> sourceColumnToViewList` is enough.

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/parameter/IntoPathDescriptor.java [hunk 2]
```diff
@@ -47,6 +49,9 @@ public class IntoPathDescriptor {
   // List<(sourceColumn, targetPath)>
   private List<Pair<String, PartialPath>> sourceTargetPathPairList;
 
+  // sourceColumn -> viewPath, if the viewPath is not empty
+  private List<Pair<String, String>> sourceColumnToViewList;
```
Comment (JackieTien97) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/parameter/IntoPathDescriptor.java:53]: It seems that `List<String> sourceColumnToViewList` is enough.

**Review by JackieTien97 — CHANGES_REQUESTED** [review]
Think more about the align by device + select into

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/parameter/IntoPathDescriptor.java (hunk 2 above)
Comment (Beyyes) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/parameter/IntoPathDescriptor.java]: fixed

**Review by JackieTien97 — APPROVED** [review]

**Pull request merged**